pom.xml

```diff
@@ -76,11 +76,31 @@
                 <version>${dependency.alfresco-transform-model.version}</version>
             </dependency>
             <!-- via Data Model / OpenCMIS Client -->
+            <dependency>
+                <groupId>org.apache.cxf</groupId>
+                <artifactId>cxf-core</artifactId>
+                <version>3.3.2</version>
```

Comment:
Should have single dependency version referenced (so that 3.3.2 can be changed in one place next time around) ?

Reply:
Yes, we should. Done.
I've also downgraded the version to **3.2.9**, so it would pass our whitesource check during the build.